MonoGame.Framework.Content.Pipeline/Processors/FontDescriptionProcessor.cs

```diff
@@ -32,33 +32,33 @@ public FontDescriptionProcessor()
         public override SpriteFontContent Process(FontDescription input, ContentProcessorContext context)
         {
             var output = new SpriteFontContent(input);
-            var fontFile = FindFont(input.FontName, input.Style.ToString());
```

Comment:
Looks like your editor is "fixing" the formatting when you are saving the file.

Reply:
Do you want this "formatting" updates reverted @tomspilman ?